recipe/bld.bat

```diff
@@ -0,0 +1,20 @@
+@echo on
+
+python tools/ci_build/build.py ^
+    --compile_no_warning_as_error ^
+    --enable_lto ^
+    --build_dir build-ci ^
+    --cmake_extra_defines EIGEN_MPL2_ONLY=ON "onnxruntime_USE_COREML=OFF" "onnxruntime_BUILD_SHARED_LIB=ON" "onnxruntime_BUILD_UNIT_TESTS=ON" CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```

Comment:
The build errors seem to be gtest related. Could you try with `onnxruntime_BUILD_UNIT_TESTS=OFF`, please?